Singularity/Singularity/Screen/ScreenClasses/GameScreen.cs

```diff
@@ -384,7 +377,14 @@ private void SettlerBuild(object sender, EventArgs eventArgs, Vector2 v, Settler
             // TODO eventually the EPlacementType should be instance but currently that
             // TODO requires a road to be place and therefore throws an exception !!!!!
 
-            var cCenter = PlatformFactory.Get(EPlatformType.Command, ref mDirector, v.X - 55, v.Y - 100);
+            // adds the command center to the GameScreen, as well as two general units
+            
+            var graphid = IdGenerator.NextiD();
+            mDirector.GetDistributionDirector.AddManager(graphid);
```

Comment:
This will not work, never add graph ids yourself, it will just make things go real bad. All graph Ids are getting set in the structure map and only there, so calling it here basically assigns a dead ID which leads to problems for UI and stuff